src/RightsService.Mappers/RoleInfoMapper.cs

```diff
@@ -0,0 +1,31 @@
+﻿using LT.DigitalOffice.RightsService.Mappers.Interfaces;
+using LT.DigitalOffice.RightsService.Models.Db;
+using LT.DigitalOffice.RightsService.Models.Dto.Models;
+using LT.DigitalOffice.RightsService.Models.Dto.Responses;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.RightsService.Mappers
+{
+    public class RoleInfoMapper : IRoleInfoMapper
+    {
+        public RoleInfo Map(DbRole value, List<RightResponse> rights, List<UserInfo> users)
+        {
+            if (value == null)
+            {
+                throw new ArgumentNullException(nameof(value));
+            }
+
+            return new RoleInfo
+            {
+                Id = value.Id,
+                Name = value.Name,
+                Description = value.Description,
+                CreatedAt = value.CreatedAt,
```

Comment:
Does user need this info? I guess this only for us, for history

Reply:
They didn't say anything about it on the call, so I think it's ok

Reply:
Ask Spartak about it